### ISSUE TITLE
[Info] Semi-working settings for Lilygo T8 ESP32-S2

### ISSUE THREAD

**Issue by Shohreh** [Bodmer/TFT_eSPI#2042]
While looking for other infos, I happened on the following page that shows settings for [that board](http://www.lilygo.cn/claprod_view.aspx?TypeId=21&Id=1322).

Using those, some scripts in TFT_eSPI's Examples folder work, others don't (screen remains dark.)

https://lecity.edu.au/?p=2553

```
//   libraries/TFT_eSPI/User_Setup.h                     
#define ST7789_DRIVER      // Full configuration option

#define TFT_WIDTH  135
#define TFT_HEIGHT 240

#define TFT_CS   34  // Chip
#define TFT_DC   37  // Data Command
#define TFT_RST  38    // Set TFT_RST
#define TFT_BL   33  // LED backlight
#define TFT_MOSI 35
#define TFT_SCLK 36

#define LOAD_GLCD   // Font 1.
#define LOAD_FONT2  // Font 2. Small 16 pixel high font,
#define LOAD_FONT4  // Font 4. Medium 26 pixel high font,
#define LOAD_FONT6  // Font 6. Large 48 pixel font,
#define LOAD_FONT7  // Font 7. 7 segment 48 pixel font,
#define LOAD_FONT8  // Font 8. Large 75 pixel font
//#define LOAD_FONT8N // Font 8. Alternative to Font 8 above,
#define LOAD_GFXFF  // FreeFonts.

#define SMOOTH_FONT

#define SPI_FREQUENCY  27000000
#define SPI_READ_FREQUENCY  20000000
#define SPI_TOUCH_FREQUENCY  2500000

#define USE_HSPI_PORT  // Needed to coexist with SD card but issues with this board !!
```

**Comment by lyusupov** [comment]
https://github.com/Bodmer/TFT_eSPI/issues/2022#issuecomment-1264568239

**Comment by Bodmer** [comment]
Solved in #2022
